src/stories/Library/Heros/hero/HeroInner.tsx

```diff
@@ -44,6 +47,12 @@ const HeroInner: React.FunctionComponent<HeroInnerProps> = ({
       {description && (
         <div className="hero__description">
           <p>{description} </p>
+
+          {arrow && (
+            <div>
```

Comment:
I am not a fan of these filler seemingly unnecessary divs.

As I understand it it is needed because we want to apply `margin-bottom` to `.hero__description > *` and we need a block element to do that. The arrow element is not a block. 

This is totally outside the scope of this PR though.